images/runtime/node/generateDockerfiles.sh

```diff
@@ -12,17 +12,39 @@ source $REPO_DIR/build/__nodeVersions.sh
 
 declare -r DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 declare -r DOCKERFILE_TEMPLATE="$DIR/template.Dockerfile"
-declare -r RUNTIME_BASE_IMAGE_NAME_PLACEHOLDER="%RUNTIME_BASE_IMAGE_NAME%"
+declare -r RUNTIME_BASE_IMAGE_NAME_PLACEHOLDER="%RUNTIME_BASE_IMAGE_NAME%-$1"
+declare -r NODE_BUSTER_VERSION_ARRAY=($NODE10_VERSION $NODE12_VERSION $NODE14_VERSION)
+
+echo "$1"
 
 cd $DIR
-for VERSION_DIRECTORY in $(find . -type d -iname '[0-9]*' -printf '%f\n')
-do
-	echo "Generating Dockerfile for image $VERSION_DIRECTORY..."
+if [ "$1" == "buster" ];then
```

Comment:
Move the content under this IF condition to a function and call it for both debian flavors

Reply:
which content?

---
In reply to: [426187878](https://github.com/microsoft/Oryx/pull/592#discussion_r426187878) [](ancestors = 426187878)